Mage.Server/src/main/java/mage/server/MageServerImpl.java

```diff
@@ -1144,13 +1141,10 @@ public void endUserSession(final String sessionId, final String userSessionId) t
     @Override
     public void removeTable(final String sessionId, final UUID tableId) throws MageException {
         execute("removeTable", sessionId, () -> {
-            Optional<Session> session = SessionManager.instance.getSession(sessionId);
-            if (!session.isPresent()) {
-                logger.error("Session not found : " + sessionId);
```

Comment:
this section removed or just handled elsewhere? not sure how important this logging was but feel like it should be retained if we're not sure.